src/client/game/round-results.js

```diff
@@ -1,22 +1,107 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
+import storageService from '../services/storage-service';
+
 import style from '../common/common.scss'
+import gameStyle from './game.scss';
 
 export class RoundResults extends Component {
-	render() {
-		const finalAnswersData = Object.keys(this.props.finalAnswers).map(id => {
+	componentWillMount(){
+		this.setState({
+			winners: {},
+			plebs: {},
+			losers: {}
+		})
+	}
+
+	componentWillReceiveProps(nextProps) {
+		let sortedKeys = Object.keys(nextProps.finalAnswers);
+		sortedKeys.sort((a,b) => {
+			return nextProps.finalAnswers[a].score - nextProps.finalAnswers[b].score;
+		});
+
+		sortedKeys.reverse();
```

Comment:
This might not be the optimal way to achieve this given you've implemented the sorting algorithm two lines above